operator-common/src/main/java/io/strimzi/operator/common/Annotations.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.ObjectMeta;
+import io.fabric8.kubernetes.api.model.PodTemplateSpec;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Annotations {
+
+    public static final String STRIMZI_DOMAIN = "strimzi.io";
+
+    public static final String ANNO_DEP_KUBE_IO_REVISION = "deployment.kubernetes.io/revision";
+
+    private static Map<String, String> putAnnotation(ObjectMeta metadata) {
```

Comment:
Why is this called `putAnnotation`? Wouldn't something like `getAnnotation` make more sense?